### ISSUE TITLE
errors from validator.w3.org

### ISSUE THREAD

**Issue by bwlang** [grimen/validatious-on-rails#29]
# Warning  Line 23, Column 153: character "&amp;" is the first character of a delimiter but occurred as data

…ue, params){return !v2.empty(value) && !(typeof value.length !== 'undefined' 

✉

This message may appear in several cases:

```
* You tried to include the "<" character in your page: you should escape it as "&lt;"
* You used an unescaped ampersand "&": this may be valid in some contexts, but it is recommended to use "&amp;", which is always safe.
* Another possibility is that you forgot to close quotes in a previous tag.
```
# Warning Line 23, Column 154: character "&amp;" is the first character of a delimiter but occurred as data

…e, params){return !v2.empty(value) && !(typeof value.length !== 'undefined' &

✉

This message may appear in several cases:

```
* You tried to include the "<" character in your page: you should escape it as "&lt;"
* You used an unescaped ampersand "&": this may be valid in some contexts, but it is recommended to use "&amp;", which is always safe.
* Another possibility is that you forgot to close quotes in a previous tag.
```
# Warning Line 23, Column 194: character "&amp;" is the first character of a delimiter but occurred as data

…typeof value.length !== 'undefined' && value.length === 0) && !/^[\s\t\n]*$/.

✉

This message may appear in several cases:

```
* You tried to include the "<" character in your page: you should escape it as "&lt;"
* You used an unescaped ampersand "&": this may be valid in some contexts, but it is recommended to use "&amp;", which is always safe.
* Another possibility is that you forgot to close quotes in a previous tag.
```
# Warning Line 23, Column 195: character "&amp;" is the first character of a delimiter but occurred as data

…ypeof value.length !== 'undefined' && value.length === 0) && !/^[\s\t\n]*$/.t

✉

This message may appear in several cases:

```
* You tried to include the "<" character in your page: you should escape it as "&lt;"
* You used an unescaped ampersand "&": this may be valid in some contexts, but it is recommended to use "&amp;", which is always safe.
* Another possibility is that you forgot to close quotes in a previous tag.
```
# Warning Line 23, Column 217: character "&amp;" is the first character of a delimiter but occurred as data

… 'undefined' && value.length === 0) && !/^[\s\t\n]*$/.test(value);},message: 

✉

This message may appear in several cases:

```
* You tried to include the "<" character in your page: you should escape it as "&lt;"
* You used an unescaped ampersand "&": this may be valid in some contexts, but it is recommended to use "&amp;", which is always safe.
* Another possibility is that you forgot to close quotes in a previous tag.
```
# Warning Line 23, Column 218: character "&amp;" is the first character of a delimiter but occurred as data

…'undefined' && value.length === 0) && !/^[\s\t\n]*$/.test(value);},message: "

✉

This message may appear in several cases:

```
* You tried to include the "<" character in your page: you should escape it as "&lt;"
* You used an unescaped ampersand "&": this may be valid in some contexts, but it is recommended to use "&amp;", which is always safe.
* Another possibility is that you forgot to close quotes in a previous tag.
```
# Error Line 23, Column > 80: XML Parsing Error: xmlParseEntityRef: no name

…/javascript">v2.Validator.add({acceptEmpty: false,fn: function(field, value, …

✉
# Error Line 23, Column > 80: XML Parsing Error: xmlParseEntityRef: no name

…/javascript">v2.Validator.add({acceptEmpty: false,fn: function(field, value, …

✉
# Error Line 23, Column > 80: XML Parsing Error: xmlParseEntityRef: no name

…/javascript">v2.Validator.add({acceptEmpty: false,fn: function(field, value, …

✉
# Error Line 23, Column > 80: XML Parsing Error: xmlParseEntityRef: no name

…/javascript">v2.Validator.add({acceptEmpty: false,fn: function(field, value, …

✉
# Error Line 23, Column > 80: XML Parsing Error: xmlParseEntityRef: no name

…/javascript">v2.Validator.add({acceptEmpty: false,fn: function(field, value, …

✉
# Error Line 23, Column > 80: XML Parsing Error: xmlParseEntityRef: no name

…/javascript">v2.Validator.add({acceptEmpty: false,fn: function(field, value, …

**Comment by grimen** [comment]
Uhm...? =)

**Comment by grimen** [comment]
What I mean with that ^ was: You post was hard to follow. I got the point though. Not high prio right now to pass the W3C validators as there are other more important bugs that needs to be fixed. (Irony: They don't seem to cope with standardizing the web anyways: moving with the speed of thousand snails LOL)
